index.js

```diff
@@ -20,7 +20,7 @@ function isFunction(f) {
 
 module.exports = {
 
-  browser: process.browser,
+  browser: typeof process === "undefined" || !process.nextTick || Boolean(process.browser),
```

Comment:
I still don't understand why `process` can be undefined...
`!!(expression)` is much faster than `Boolean(expression)`